third_party/2and3/redis/client.pyi

```diff
@@ -161,22 +163,22 @@ class Redis(object):
     def type(self, name): ...
     def watch(self, *names): ...
     def unwatch(self): ...
-    def blpop(self, keys, timeout=...): ...
-    def brpop(self, keys, timeout=...): ...
+    def blpop(self, keys: Union[_Str, Sequence[_Str]], timeout: int = ...) -> Optional[Tuple[bytes, bytes]]: ...
+    def brpop(self, keys: Union[_Str, Sequence[_Str]], timeout: int = ...) -> Optional[Tuple[bytes, bytes]]: ...
```

Comment:
`Sequence` -> `Iterable`, since it's more general.

Reply:
Indeed. Fixed in the latest push to this branch.